docs/conandata_yml_format.md

```diff
@@ -0,0 +1,228 @@
+
+```yaml
+patches:
+  "1.2.0":
+    - patch_file: "patches/1.2.0-002-link-core-with-find-library.patch"
+      base_path: "source_subfolder"
+      url: "https://a-url-to-a-pull-request-mail-list-topic-issue-or-question"
+      sha256: "qafe4rq54533qa43esdaq53ewqa5"
+      patch_type: "portability"
+      patch_source: ...
+      patch_description: "Link CoreFoundation and CoreServices with find_library"
+```
+
+### Patches fields
+
+#### patch_file
+
+Patch file might be committed to the conan-center-index, near to the conanfile (usually, into the `patches` sub-directory). Such patch files usually have either `.diff` or `.patch` extension.
+The recommended way to generate such patches is [git format-patch](https://git-scm.com/docs/git-format-patch). The path to the patch is relative to the directory containing `conandata.yml` and `conanfile.py`.
+
+#### base_path
+
+Specifies a sub-directory in project's sources to apply patch. This directory is relative to the [source_folder](https://docs.conan.io/en/latest/reference/conanfile/attributes.html?highlight=source_folder#source-folder). Usually, it would be a `source_subfolder`, but could be a lower-level sub-directory (e.g. if it's a patch for a submodule).
+
+#### url
+
+The patch could be taken from the existing resource (e.g. some patches are distributed alongside the source tarball). See `url` section of the `sources`.
+
+#### sha256
+
+This is the hash for the patch itself, in the same way this field is used in the `sources` section.
+
+#### sha1
+
+This is the hash for the patch itself, in the same way this field is used in the `sources` section.
+
+#### md5
+
+This is the hash for the patch itself, in the same way this field is used in the `sources` section.
+
+#### patch_type
+
+The `patch_type` field specifies the type of the patch. In conan-center-index we currently accept only several kind of patches:
+
+##### official
+
+`patch_type: official` indicates the patch is distributed with source code itself. usually, this happens if release managers failed to include a critical fix to the release, but it's too much burden for them to make a new release just because of that single fix.
+[example](https://www.boost.org/users/history/version_1_72_0.html) (notice the `coroutine` patch)
+
+##### vulnerability
+
+`patch_type: vulnerability`: Indicates a patch that addresses the security issue. The patch description
+should include the index of CVE or CWE the patch addresses.
+Usually, original library projects do new releases fixing vulnerabilities for this kind of issues, but in some cases they are either abandoned or inactive.
+
+##### backport
+
+`patch_type: backport`: Indicates a patch that backports an existing bug fix from the newer release or master branch (or equivalent, such as main/develop/trunk/etc). The patch source may be a pull request, or bug within the project's issue tracker.
+Backports are accepted only for bugs, but not for feature requests.
```

Comment:
```suggestion
Backports are accepted only for bugs that break normal execution flow, never for feature requests.
```

This is what we called _broken execution_, it is not just a bug people might be relying on its _buggy_ behavior